Extension/package.json

```diff
@@ -6728,7 +6728,7 @@
         "generate-options-schema": "ts-node -T ./.scripts/generateOptionsSchema.ts",
         "copy-walkthrough-media": "ts-node -T ./.scripts/copyWalkthruMedia.ts",
         "translations-export": "yarn install && yarn prep && yarn generate-native-strings && gulp translations-export",
-        "translations-generate": "set NODE_OPTIONS=--no-experimental-fetch && gulp translations-generate",
+        "translations-generate": "gulp translations-generate",
```

Comment:
This option fails my build when I install the latest npm. Does anyone know what it was for? It seems to work without it.

Reply:
It looks like it was added here: https://github.com/microsoft/vscode-cpptools/pull/10771